src/main/java/Utils.java

```diff
@@ -0,0 +1,37 @@
+import java.util.List;
+
+public class Utils {
+    /**
+     * Wraps replies with horizontal lines and indentation.
+     */
+    public static void reply(List<String> messages) {
+        String allMessages = "";
+        String indentation = "     ";
+        for (String message : messages) {
+            if (message.isEmpty()) {
+                continue;
+            }
+            allMessages += indentation + message + "\n";
+        }
+        System.out.println(String.format("%s\n%s%s\n",
+                Constants.horizontalLine,
+                allMessages,
+                Constants.horizontalLine
+        ));
```

Comment:
perhaps it is better to put )) at the previous line? I am not too sure about this actually.